### ISSUE TITLE
Mark indices as APM

### ISSUE THREAD

**Issue by graphaelli** [elastic/apm-server#1802]
Since index names may be endlessly customized, make it possible to consistently identify an APM index.  This will also help the migration assistant know which indices need to be / have already been migrated.

https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-meta-field.html seems to be the right way to do this.

**Comment by simitt** [comment]
@jalvz isn't this done already?

**Comment by jalvz** [comment]
Yes, apparently https://github.com/elastic/apm-server/pull/1862#issuecomment-459277204 didn't close the issue.
Thanks
